src/main/java/org/ethereum/beacon/discovery/schema/NodeRecordFactory.java

```diff
@@ -62,6 +62,10 @@ public NodeRecord fromBase64(String enrBase64) {
     return fromBytes(Base64.getUrlDecoder().decode(enrBase64));
   }
 
+  public NodeRecord fromEnr(String enr) {
+    return fromBase64(enr.substring("enr:".length()));
```

Comment:
I'd probably keep the `enr:` prefix as optional (so do the `.startsWith("enr:')` check like Teku does. Otherwise we'll get a really cryptic error message about invalid base64 if the prefix is missing.

Reply:
I understand. I did the check like Teku does.